### PR TITLE
add parrot

### PR DESCRIPTION
**Pull request by daliya90**
Please add my species.

### CODE REVIEW

File: params/parrot.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.25
```
Comment (DaliyaAD) [inline params/parrot.R:7]: please assign a number between 0 and 5 to this variable

**Review by DaliyaAD — CHANGES_REQUESTED** [review]
good work Please have a look at my comments

**Comment by daliya90** [comment]
Resolved

**Comment by daliya90** [comment]
please add my changes

**Pull request closed without merging**